report_thesis/src/sections/problem_definition.tex

```diff
@@ -1,7 +1,7 @@
 \section{Problem Definition}\label{sec:problem_definition}
-The primary objective of this research is to enhance computational methods for the accurate and robust quantification of chemical compositions using \gls{libs} data.
+The primary objective of this research is to systematically evaluate existing computational methods to identify the best-performing techniques and subsequently develop an optimized machine learning pipeline for accurately and robustly quantifying the chemical compositions of Martian geological samples using LIBS spectral data.
```

Comment:
Now that I return to this one, perhaps we've regressed slightly?
The objective contains action items, which may not be ideal.
Ideally, we'd talk about the problem throughout this section, and only at the end start defining _how_ we'd propose achieving it.